Create hire_the_most_curious_ones.md

zappabruno opened the pull request.

zappabruno committed:
Create hire_the_most_curious_ones.md

zappabruno commented:
@jlee16 can you see the file I posted? is the PR rightly done this time? thanks!

jlee16 commented:
I can definitely see the file you posted.  I think the pull request is done right too, but I can't 100% be sure yet since I also need to play around with GitHub more to figure it out.  But I can see that your branch exists and what you wrote etc.  Also I feel like there is an additional fork that happened when you created this particular branch but I'm not 100% sure about that either.  Will check in more detail when I have more time

zappabruno commented:
Cool. 
I'm still not 100% sure how this GitHub works.
But what I think I did was:
1. initially created a branch off your fork/repository and added a file to it.
2. tried to set a PR, but it was being posted in the classes repository/fork. So I closed the PR, but kept the file.
3. I forked from your jlee16/lessons-learned and created zappabruno/lessons-learned 
4.  in there I re-created the file and asked for a PR, and made the comment above. 

I wonder if after getting your feedback, and adjusting the file, when I merge it will go to your fork/repository, and then when you merge it, all of our teams files will go to the class repository

zappabruno commented:
@saadahmed1 are you able to see and comment on this PR?  thanks!

zappabruno committed:
Update hire_the_most_curious_ones.md

Added: "in addition to looking for good references/credentials/relevant past experience"

zappabruno commented:
@saadahmed1 Can you have a look at the change I've made in the first line, and let me know what you think?  

Added: In addition to looking for good references/credentials/relevant past experience

saadahmed1 commented:
@zappabruno Yes. looks good !
